kivy/uix/video.py

```diff
@@ -31,21 +42,51 @@ def on_duration_change(instance, value):
 from kivy.core.video import Video as CoreVideo
 from kivy.resources import resource_find
 from kivy.properties import (BooleanProperty, NumericProperty, ObjectProperty,
-                             OptionProperty)
+                             OptionProperty, StringProperty)
 
 
 class Video(Image):
     '''Video class. See module documentation for more information.
     '''
 
+    source = StringProperty(None)
+    '''Filename / source of your video.
+
+    :attr:`source` is a :class:`~kivy.properties.StringProperty` and
+    defaults to None.
+
+    .. versionchanged:: 2.1.0
+        While the use of ``source`` is the historical way for defining
+        video source and stays supported for backward compatibility, it is
+        recommended to use ``video_source`` instead.
+    '''
+
+    video_source = StringProperty(None)
+    '''Filename / source of your video.
+
+    :attr:`video_source` is a :class:`~kivy.properties.StringProperty` and
+    defaults to None.
+
+    .. versionadded:: 2.1.0
+    '''
+
+    preview_source = StringProperty(None)
+    '''Filename / source of a preview image displayed before video starts.
+
+    :attr:`preview_source` is a :class:`~kivy.properties.StringProperty` and
+    defaults to None.
```

Comment:
```suggestion
    defaults to None. If None/empty string and :attr:`source ` is a
    filename, that will be used for the preview filename.
    Otherwise, :attr:`source` will be set to :attr:`preview_source`.
```